src/mapping_igvc/package.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0"?>
```

Comment:
Same comment as the CMakelists.txt for removing templated entries

Reply:
Ok, roughly the same sort of reply here as with the `CMakeLists.txt`, I've made another note in the #software-leads channel. Once we decide on things there we can always come back and nuke some (or all) comments.

Reply:
After some feedback, looks like we should keep things as they are, leaving comments in for the portions of the `CMakeLists.txt` that we're using.